packages/babel-parser/test/fixtures/es2015/statements/label-invalid-let/output.json

```diff
@@ -2,7 +2,7 @@
   "type": "File",
   "start":0,"end":13,"loc":{"start":{"line":1,"column":0,"index":0},"end":{"line":1,"column":13,"index":13}},
   "errors": [
-    "SyntaxError: Missing semicolon. (1:8)"
+    "SyntaxError: Lexical declaration cannot appear in a single-statement context. (1:5)"
```

Comment:
The input is
```js
foo: let bar;
```

Now we throw a better error here.

Reply:
Yes, this improves the parsed AST result at the same time.